### ISSUE TITLE
Is there a reason TransportQueryError doesn't extend TransportError?

### ISSUE THREAD

**Issue by helderco** [graphql-python/gql#391]
I expected all exceptions to be captured by `gql.exceptions.TransportError`:

```python
from gql.exceptions import TransportError

try:
    ...
except TransportError as e:
    ...
```

But `TransportQueryError` extends from `Exception`:

https://github.com/graphql-python/gql/blob/2827d887db4c6951899a8e242af55863328f68a2/gql/transport/exceptions.py#L30

Is this by design?

**Comment by leszekhanusz** [comment]
No, that's not by design, it's a bug.
Thank you for reporting this!

**Comment by leszekhanusz** [comment]
Fixed in [release 3.5.0b2](https://github.com/graphql-python/gql/releases/tag/v3.5.0b2).
